Add models and datatypes to support the Device configuration feature set (#17)
AMWA-TV/nmos-control-feature-sets: device-configuration/README.md

> # NMOS Control Feature Sets: Device configuration

Author: Yes I believe that is the case and then the render action will render the table once we merge to main.

    attribute NcPropertyRestoreNoticeType    noticeType;    // Property restore notice type
    attribute NcString    noticeMessage;                    // Property restore notice message
};
```

### NcObjectPropertiesSetValidation

```typescript
interface NcObjectPropertiesSetValidation {
    attribute NcRolePath    path;                                   // Object role path
    attribute NcRestoreValidationStatus    status;                  // Validation status
    attribute sequence<NcPropertyRestoreNotice>    notices;         // Validation property notices
    attribute NcString?    statusMessage;                           // Validation status message
};
```

### NcMethodResultBulkValuesHolder

```typescript
interface NcMethodResultBulkValuesHolder: NcMethodResult {
    attribute NcBulkValuesHolder    value; // Bulk values holder value
};
```

### NcMethodResultObjectPropertiesSetValidation

```typescript
interface NcMethodResultObjectPropertiesSetValidation: NcMethodResult {
    attribute sequence<NcObjectPropertiesSetValidation>    value; // Object properties set path validations
};
```

## Control classes

### NcBulkPropertiesManager

> The `BulkPropertiesManager` offers a central model for getting and setting properties of multiple role paths.
> It also allows pre-validation of a data set before attempting to use in setting multiple properties of multiple role paths.

Reviewer: Possible rewording for clarification?

```suggestion
The `BulkPropertiesManager` offers a central model for getting and setting multiple properties on multiple role paths.
It also allows pre-validation of a data set before attempting these get and set operations.
```
